admin/languages.php

```diff
@@ -139,15 +139,16 @@
         }
 
 // create additional orders_status records
-        $orders_status = $db->Execute("SELECT orders_status_id, orders_status_name
+        $orders_status = $db->Execute("SELECT orders_status_id, orders_status_name, sort_order
                                        FROM " . TABLE_ORDERS_STATUS . "
                                        WHERE language_id = " . (int)$_SESSION['languages_id']);
 
         foreach ($orders_status as $status) {
-          $db->Execute("INSERT INTO " . TABLE_ORDERS_STATUS . " (orders_status_id, language_id, orders_status_name)
+          $db->Execute("INSERT INTO " . TABLE_ORDERS_STATUS . " (orders_status_id, language_id, orders_status_name, sort_order)
                         VALUES ('" . (int)$status['orders_status_id'] . "',
                                 '" . (int)$insert_id . "',
-                                '" . zen_db_input($status['orders_status_name']) . "')");
+                                '" . zen_db_input($status['orders_status_name']) . "',
+                                '" . (int)$status['sort_order'] . "')");
```

Comment:
Single quotes around integer values for a database field are neither needed nor wanted and the casting is redundant as you've just pulled the information from the database.

Reply:
You are right, I stupidly reproduced original formatting... I changed that for the 3 integer values.